### DISCUSSION TITLE
Change state-label

### DISCUSSION THREAD

**Discussion by Rainier55** [audiconnect/audi_connect_ha#295] [Q&A]
Dear Team,
where can i change state-label of binary_sensor.audi_q3_xxxxx_windows from "Geschlossen" eg. to "Zu". I can't find it. Reason: in Smartphone-sight the text is overlapping. 
![Audi_Q3](https://github.com/audiconnect/audi_connect_ha/assets/7571285/0d5ccdfa-4cf7-4674-b689-2a827899688b)

**Comment by Kolbi** [comment]
I guess we are using here the standard Home-Assistant states... https://www.home-assistant.io/integrations/binary_sensor/ I don't know if there is another way to shorten them.

Maybe there is a way to do it directly in the picture-elements object?
